05.ls/ls.rb

```diff
@@ -31,13 +31,13 @@ def main
 
 def output_for_long_format(files)
   output_total_block_count(files)
-  output_files_l(files)
+  long_formatted_output_files(files)
```

Comment:
メソッド名は動詞で始めた方がいいですね。
output_long_formatted_filesのような名前であれば英語的にも自然です。

別のアプローチとして、ここにメソッドの内容をがっと移動させて、メソッド作成を避ける、という方法も考えられます（rubocopの警告が出ないなら）。

```suggestion
  files.each do |file|
    stat_file = File.stat(file)
    output_data = [
      FILE_TYPE[stat_file.ftype] + rewrite_permission(stat_file),
      FILE_TYPE[stat_file.ftype] + build_permission(stat_file),
      stat_file.nlink,
      Etc.getpwuid(stat_file.uid).name,
      Etc.getgrgid(stat_file.gid).name,
      stat_file.size.to_s.rjust(4),
      stat_file.mtime.strftime('%b %d %H:%M'),
      file
    ]
    puts output_data.join(' ')
  end
```

どちらにするかはお任せしますー。

Reply:
`output_for_long_format(files)`にメソッドの内容を移動させてみたところrubocopの警告は出ませんでした。
今回はご指摘頂いた２つの変数名を修正する形で提出しましたが、↑の方法でもOKということを覚えておきます！